INFO - 2024-10-20 10:01:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-20 10:01:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-10-20 10:02:08 --> CodeIgniter\Validation\Exceptions\ValidationException: min_lenght não é uma regra válida.
[Method: POST, Route: maquinario/onSave]
in SYSTEMPATH\Validation\Validation.php on line 345.
 1 SYSTEMPATH\Validation\Validation.php(345): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('min_lenght')
 2 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('nome', 'nome', 'Grade niveladora', [...], [...], 'nome')
 3 SYSTEMPATH\BaseModel.php(1577): CodeIgniter\Validation\Validation->run([...], null, null)
 4 SYSTEMPATH\BaseModel.php(809): CodeIgniter\BaseModel->validate([...])
 5 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\MaquinarioController.php(47): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\MaquinarioController->onSave()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MaquinarioController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\VTornos\\public\\index.php')
CRITICAL - 2024-10-20 10:03:21 --> CodeIgniter\Validation\Exceptions\ValidationException: min_lenght não é uma regra válida.
[Method: POST, Route: maquinario/onSave]
in SYSTEMPATH\Validation\Validation.php on line 345.
 1 SYSTEMPATH\Validation\Validation.php(345): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('min_lenght')
 2 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('nome', 'nome', 'Grade niveladora', [...], [...], 'nome')
 3 SYSTEMPATH\BaseModel.php(1577): CodeIgniter\Validation\Validation->run([...], null, null)
 4 SYSTEMPATH\BaseModel.php(809): CodeIgniter\BaseModel->validate([...])
 5 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\MaquinarioController.php(47): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\MaquinarioController->onSave()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MaquinarioController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\VTornos\\public\\index.php')
CRITICAL - 2024-10-20 10:06:27 --> CodeIgniter\Validation\Exceptions\ValidationException: max_lenght não é uma regra válida.
[Method: POST, Route: maquinario/onSave]
in SYSTEMPATH\Validation\Validation.php on line 345.
 1 SYSTEMPATH\Validation\Validation.php(345): CodeIgniter\Validation\Exceptions\ValidationException::forRuleNotFound('max_lenght')
 2 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('nome', 'nome', 'Grade niveladora', [...], [...], 'nome')
 3 SYSTEMPATH\BaseModel.php(1577): CodeIgniter\Validation\Validation->run([...], null, null)
 4 SYSTEMPATH\BaseModel.php(809): CodeIgniter\BaseModel->validate([...])
 5 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\MaquinarioController.php(47): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\MaquinarioController->onSave()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MaquinarioController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\VTornos\\public\\index.php')
ERROR - 2024-10-20 10:06:49 --> Error connecting to the database: mysqli_sql_exception: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente in C:\xampp\htdocs\VTornos\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\xampp\htdocs\VTornos\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('localhost', 'root', '', 'vtornos', 3306, '', 0)
#1 C:\xampp\htdocs\VTornos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\VTornos\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\VTornos\system\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ma...', Array, false)
#4 C:\xampp\htdocs\VTornos\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\VTornos\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\VTornos\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\VTornos\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\VTornos\app\Controllers\MaquinarioController.php(47): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\VTornos\system\CodeIgniter.php(933): App\Controllers\MaquinarioController->onSave()
#10 C:\xampp\htdocs\VTornos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MaquinarioController))
#11 C:\xampp\htdocs\VTornos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\VTornos\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\VTornos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\VTornos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xampp\htdocs\VTornos\system\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs...')
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente in C:\xampp\htdocs\VTornos\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\VTornos\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\VTornos\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\VTornos\system\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ma...', Array, false)
#3 C:\xampp\htdocs\VTornos\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\VTornos\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\VTornos\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#6 C:\xampp\htdocs\VTornos\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#7 C:\xampp\htdocs\VTornos\app\Controllers\MaquinarioController.php(47): CodeIgniter\BaseModel->save(Array)
#8 C:\xampp\htdocs\VTornos\system\CodeIgniter.php(933): App\Controllers\MaquinarioController->onSave()
#9 C:\xampp\htdocs\VTornos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MaquinarioController))
#10 C:\xampp\htdocs\VTornos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\VTornos\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\VTornos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\VTornos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\VTornos\system\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2024-10-20 10:06:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente
[Method: POST, Route: maquinario/onSave]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `maquinario` (`nome`) VALUES (:nome:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\MaquinarioController.php(47): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\MaquinarioController->onSave()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MaquinarioController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\VTornos\\public\\index.php')
INFO - 2024-10-20 10:07:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-20 10:18:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-20 10:18:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-20 10:18:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
